src/main/resources/data/games/strategy/triplea/help/tripleaForum.html

```diff
@@ -1,15 +0,0 @@
-<html>
```

Comment:
For some reason, this file wasn't covered by #1564 (the corresponding PR may have been in-flight when #1564 was opened).  Anyway, that's why it was moved out of the `data` folder.